src/ansi-c/expr2c.cpp

```diff
@@ -2914,6 +2914,12 @@ std::string expr2ct::convert_code(
   if(statement=="unlock")
     return convert_code_unlock(src, indent);
 
+  if(statement==ID_atomic_begin)
+    return indent_str(indent)+"__CPROVER_atomic_begin();";
```

Comment:
Here I think you can use `CPROVER_PREFIX` as opposed to `__CPROVER`

Reply:
Throughout the file (expr2c.cpp)  "__CPROVER_" is being used as a prefix, for the sake of consistency I think it is better to stick with "__CPROVER_"

Reply:
Good point. Leave it as is, I've created #1758 to clean this up at a later point.